cookiecutter.json

```diff
@@ -22,5 +28,19 @@
     "has_spring_boot_rest": "n",
     "has_spring_boot_session": "n",
     "java_version": "1.8",
-    "spring_boot_version": "2.1.9.RELEASE"
+    "spring_boot_version": "2.1.9.RELEASE",
+  
+  	"xhr_option": ["fetch", "axios", "superagent", "request", "none"], 
+    "has_formik": "n",
+    "state_management_option": ["redux", "mobx3", "mobx4", "mobx5", "none"],
+    "has_styledComponents": "n",
+    "has_materialUi": "n",
+    "pm_option": ["npm", "yarn"],
+    "pm_config_file": "",
```

Comment:
@driverdan this is necessary in case the user specifies their own config file. However, I don't know how we'll even get a user-specified config file into the codebase of a generated app. So, this option might be moot.